js/main.js

```diff
@@ -162,6 +168,33 @@ function appendingCardImageToDeckURL() {
   }
 }
 
+function resetSearchResults() {
+  while ($searchResultFeed.firstChild) {
+    $searchResultFeed.removeChild($searchResultFeed.firstChild);
+  }
+}
+
+function resetDeckResults() {
+  while ($deckList.firstChild) {
+    $deckList.removeChild($deckList.firstChild);
+  }
+}
```

Comment:
Instead of duplicating this behavior, I would make one function with an `element` parameter which is the element to remove the children of.